src/js/abstract/is-same-node-once.js

```diff
@@ -0,0 +1,39 @@
+const TEXT_NODE = 3;
+let sameNodeCache = [];
+
+/**
+ * Make sure that another piece of code is/can managing that part of the DOM tree.
+ *
+ * @link https://github.com/choojs/nanomorph#caching-dom-elements
+ * @param node
+ */
+export function isSameNodeOnce(node) {
+  return;
+  // @todo: react text-node updates break if we do this!
+  if (node.nodeType === TEXT_NODE) {
```

Comment:
@LucaMele 
This check did **NOT** fixed the broken accordion and header.
So not an option and the whole function seems to stay pointless.
Cause I really don't know yet what is causing our issue and why my fix works I'm uncertain if should delete it now...

Reply:
`isSameNodeOnce` did work for Custom-elements only.
But broke for integration with react.
So may we could say that `deepClone` and `isSameNode` optimisation are configurable?